angular/projects/spark-angular/src/lib/components/inputs/Textarea.stories.ts

```diff
@@ -112,27 +112,30 @@ invalidTextarea.story = {
     jest: [
       'sprk-textarea-container.component',
       'sprk-field-error.directive',
-      'sprk-input.directive',
+      'sprk-textarea.directive',
       'sprk-label.directive',
     ],
-  }
+  },
 };
 
 export const disabledTextarea = () => ({
   moduleMetadata: modules,
   template: `
     <sprk-textarea-container>
       <label
-        class="sprk-b-Label--disabled"
+        isDisabled="true"
         for="textarea-1"
-        sprkLabel>Description</label>
+        sprkLabel
+      >
+        Description
+      </label>
       <textarea
         name="textarea_input"
         id="textarea-1"
         [(ngModel)]="textarea_input"
         #textareaInput="ngModel"
         data-id="textarea-1"
-        sprkInput
+        sprkTextarea
```

Comment:
the sprkTextarea directive has an idString input we should use here.

Reply:
not sure why this isn't marked as "outdated" but it's def updated: 
![image](https://user-images.githubusercontent.com/4342363/113910588-00948c80-97a7-11eb-984c-aa48034d0b28.png)